extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -117,6 +125,45 @@ export class Bind {
      */
     this.digestQueuedAfterScan_ = false;
 
+    /**
+     * @const @private {!Array<Node>}
+     */
+    this.dynamicRoots_ = [];
```

Comment:
Continuing from https://github.com/ampproject/amphtml/pull/7602#discussion_r101837344: so the observer callback is invoked for each mutated node in addition to the target? Seems unexpected from reading the API docs.